dist/spec.json

```diff
@@ -13,7 +13,7 @@
       "name": "Apache-2.0",
       "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
     },
-    "version": "2.5.0"
+    "version": "2.6.0"
```

Comment:
Revert the version

Reply:
But spec.yaml in master branch has 2.6.0 already https://github.com/okta/okta-management-openapi-spec/blob/master/dist/spec.yaml#L28
So, what's the correct version for this change?

Reply:
2.6.1 I guess, since it's just a minor fix

Reply:
Updated version